preview-screenshots/src/test/java/io/github/droidkaigi/confsched2022/PreviewScreenshotTests.kt

```diff
@@ -0,0 +1,79 @@
+package io.github.droidkaigi.confsched2022
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.unit.Density
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.androidHome
+import app.cash.paparazzi.detectEnvironment
+import com.airbnb.android.showkase.models.Showkase
+import com.airbnb.android.showkase.models.ShowkaseBrowserComponent
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameter.TestParameterValuesProvider
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+class ComponentPreview(
+    private val showkaseBrowserComponent: ShowkaseBrowserComponent
+) {
+    val content: @Composable () -> Unit = showkaseBrowserComponent.component
+    override fun toString(): String = showkaseBrowserComponent.componentKey
+}
+
+@RunWith(TestParameterInjector::class)
+class PreviewScreenshotTests {
+
+    object PreviewProvider : TestParameterValuesProvider {
+        override fun provideValues(): List<ComponentPreview> =
+            Showkase.getMetadata().componentList.map(::ComponentPreview)
+    }
+
+    enum class BaseDeviceConfig(
+        val deviceConfig: DeviceConfig,
+    ) {
+        NEXUS_5(DeviceConfig.NEXUS_5),
+        PIXEL_5(DeviceConfig.PIXEL_5),
+        PIXEL_C(DeviceConfig.PIXEL_C),
```

Comment:
[nit] How about consider to adding the [Pixel6](https://github.com/cashapp/paparazzi/blob/master/paparazzi/paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt#L452) or [Nexus 10](https://github.com/cashapp/paparazzi/blob/master/paparazzi/paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt#L164) ? The Pixel6 is latest device, The Nexus 10 is a one of the tablet devices(I'm not sure we are suppressed to support fro Table devices)

Reply:
oh, sorry. I never know the Pixel C is tablet device...! too old device!